tests/playTests.sh

```diff
@@ -785,6 +792,7 @@ println "world!" > world.tmp
 cat hello.tmp world.tmp > helloworld.tmp
 zstd -c hello.tmp > hello.zst
 zstd -c world.tmp > world.zst
+zstd -c hello.tmp world.tmp > helloworld.zst
```

Comment:
In order for this to be a useful test, you would want to decompress it and diff it. E.g. add the lines:

```
zstd -dc helloworld.zst > result.tmp
$DIFF helloworld.tmp result.tmp
```

Reply:
ok，I'll try it.